amplifycli_react_tutorial.md

```diff
@@ -1,288 +1 @@
-      body: {
-        id: '1',
-        name: 'hello amplify!'
-      }
-    });
-    alert(JSON.stringify(response, null, 2));
-  }
-  get = async () => {
-    console.log('calling api');
-    const response = await API.get('myapi', '/items/object/1');
-    alert(JSON.stringify(response, null, 2));
-  }
-  list = async () => {
-    console.log('calling api');
-    const response = await API.get('myapi', '/items/1');
-    alert(JSON.stringify(response, null, 2));
-  }
-```
-
-Update the `render()` method to include calls to the following methods:
-
-```
-  render() {
-    return (
-      <div className="App">
-        <p> Pick a file</p>
-        <input type="file" onChange={this.uploadFile} />
-        <button onClick={this.post}>POST</button>
-        <button onClick={this.get}>GET</button>
-        <button onClick={this.list}>LIST</button>
-        
-        <S3Album level="private" path='' />
-      </div>
-    );
-  }
-```
-
-Save the file and run `amplify publish`. After the API is deployed along with the Lambda function and database table, your app is built and updated in the cloud. You can then add a record to the database by choosing **POST**, and then using **GET** or **LIST** to retrieve the record, which has been hard coded in this simple example.
-
-In your project directory, open `./amplify/backend/function` and you'll see the Lambda function that you created. The `app.js` file runs the Express function and all of the HTTP method routes are available for you to manipulate. For example, the `API.post()` in your React app corresponded to the `app.post(path, function(req, res){...})` code in this Lambda function. If you choose to customize the Lambda function, you can update it in the cloud using `amplify push`. 
-
-
-# Testing Your Function
-
-Amplify CLI supports local testing of Lambda functions. Run `amplify status` to get the resource name of the Lambda function created earlier, and then run the following:
-
-```
-amplify function invoke <resourcename>
-```
-
-In this case, the function runs, but it doesn't exit because this Lambda example starts an Express server which you need to manually close when testing it from the CLI. Use `ctrl-c` to close and open the `./amplify/backend/function/resourcename` directory to see the local structure that is packaged for Lambda invocation from API Gateway. The Lambda function is inside the `src` directory, along with `event.json`, which is used for the `amplify function invoke` command you just ran. `index.js` is also in this directory, which is the main entry point for the Serverless Express library that echoed out the test event and instantiated the server inside `app.js`. Since the Express routes defined in `app.js` don't have a path that's called via the test event, it responded with a 404 message. For more information, see https://github.com/awslabs/aws-serverless-express.
+<meta http-equiv="refresh" content="0; URL='https://github.com/aws-amplify/docs/blob/master/js/react.md'" />
```

Comment:
This should change to redirecting to the actual GH site:

https://aws-amplify.github.io/aws-amplify/docs/js/react